scripts/Diffraction/isis_powder/polaris_routines/polaris_algs.py

```diff
@@ -9,7 +9,7 @@
 
 
 def calculate_absorb_corrections(ws_to_correct, multiple_scattering):
-    mantid.MaskDetectors(ws_to_correct, SpectraList=list(range(0, 55)))
+    mantid.MaskDetectors(ws_to_correct, SpectraList=list(range(1, 55)))
```

Comment:
Is this supposed to cover less range after the change? Before it was 0->55 (length 54), now it's 1->55 (length 53)?